src/main/java/com/aws/greengrass/util/platforms/unix/linux/CgroupSubSystemV2.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.util.platforms.unix.linux;
+
+public enum CgroupSubSystemV2 {
+    Memory("", ""), CPU("", ""), Freezer("", ""),
+    Unified("","");
```

Comment:
I think we should have a better way to represent this than just a bunch of empty strings.